footil/tests/test_log.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import logging
+
+from footil import log
+
+
+class TestLog(unittest.TestCase):
+    """Class to test stdout/stderr helpers."""
+
+    def test_log_1(self):
+        """Capture output from print."""
+        res = log.capture_output(print, args=('test',))
+        self.assertEqual(res, 'test\n')
+
+    # TODO: There might be a bug with logging, so disabled tests for
+    # now.
```

Comment:
What kind of bug did you find? It comes from standard logging?

Reply:
https://bugs.python.org/issue36193

Though according to python devs, it is not a bug. It is intended to work this way. But then it complicates stderr redirection, because I need to explicitly call logging to specify what kind of stderr to use (where with print, it does rely on stderr directly where it is directed - thus why I can easily capture prints, but not logging).